pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -361,10 +401,29 @@ func loadIPSecKeys(r io.Reader) (uint8, error) {
 		ipSecKey.Spi = spi
 
 		if len(s) == 6+offset {
+			if ipSecKeysGlobal[s[5+offset]] != nil {
+				oldSpi = ipSecKeysGlobal[s[5+offset]].Spi
+			}
 			ipSecKeysGlobal[s[5+offset]] = ipSecKey
 		} else {
+			if ipSecKeysGlobal[""] != nil {
+				oldSpi = ipSecKeysGlobal[""].Spi
+			}
 			ipSecKeysGlobal[""] = ipSecKey
 		}
+
+		scopedLog.WithError(err).Warning("newtimer: oldSPI %u new spi %u", oldSpi, ipSecKey.Spi)
```

Comment:
Is this warning message in a form that you expect users to be able to react to? It seems a little cryptic at a glance. Is this a warning type situation or merely informational?

Reply:
Two more comments:
* There is an "spi" field inherited from the `scopedLog` above, reading the logs it is likely not to be obvious what that represents vs. these other two spis here
*  Log messages should avoid placing variable content into the message field, the new/old SPIs should be reported through the standard mechanism (ie `WithFields(...)`).

Reply:
That warning snuck in should be removed.

Reply:
Its not really a warning I was just debugging earlier and forgot to remove it. In fact its not even a timer anymore so newtimer also is nonsense.